src/index.html

```diff
@@ -8,10 +8,179 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <script type="text/javascript" src="./main.js">
+    </script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="fieldset">
+        <legend>
+          Personal information
+        </legend>
+        <label class="form_field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form_field">
+          Name:
+          <input type="text"
+            name="name"
+            autocomplete="off"
+            class="field"
+          >
+        </label>
+        <label class="form_field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            class="field"
+          >
+        </label>
+        <label class="form_field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            class="field"
+          >
+        </label>
+        <label class="form_field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            required=""
+            name="accept"
+            class="field"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
```

Comment:
```suggestion
      </fieldset>

      <fieldset class="fieldset">
```

Add blank line between multiple sibling blocks
Check all such cases